--- a/configure.ac
+++ b/configure.ac
@@ -200,9 +200,9 @@ AC_COMPILE_IFELSE(
                    [[int foo = TCP_USER_TIMEOUT;]])],
   iperf3_cv_header_tcp_user_timeout=yes,
   iperf3_cv_header_tcp_user_timeout=no))
-if test "x$iperf3_cv_header_tcp_user_timeout" = "xyes"; then
-    AC_DEFINE([HAVE_TCP_USER_TIMEOUT], [1], [Have TCP_USER_TIMEOUT sockopt.])
-fi
+#if test "x$iperf3_cv_header_tcp_user_timeout" = "xyes"; then
+#    AC_DEFINE([HAVE_TCP_USER_TIMEOUT], [1], [Have TCP_USER_TIMEOUT sockopt.])
+#fi
 
 # Check for IPv6 flowlabel support (believed to be Linux only)
 # We check for IPV6_FLOWLABEL_MGR in <linux/in6.h> even though we